### PR TITLE
Use short affiliations in AUTHORS.md 

### LINKED ISSUE #897: Short affiliation names in AUTHORS.md
Initial discussed in https://github.com/GenericMappingTools/pygmt/pull/857#pullrequestreview-588235476:

> > 1. Do we want the affiliation to be the full Department, University, City, Country format? I understand that the full affiliation would be needed for a paper, but again, a lot can happen between now and the paper coming out. Right now it's only the University name listed on Zenodo (hover over the names at https://zenodo.org/record/4253459).
> 
> Is the motivation to update this less often? We could forgo affiliations here and just ask for them when making releases or just when writing papers?

> Yeah, would be best to keep changes to a minimum. Do people update their ORCID page often? I suppose we could just use what's on their ORCID page, even if it might be a pain during releases to check through every single person's current location.

The full affiliation names sometimes are longer than one line in GitHub web UI, and more than 200 characters in the plaintext markdown file (see the screenshot below). 

image

And the full affiliation names won't be used unless when we write the first paper. So I prefer to have shorter affiliation names.

### PR DESCRIPTION
**Pull request by core-man**
**Description of proposed changes**

Use short affiliations instead of the full ones in `AUTHORS.md`, because the full affiliations sometimes are longer than one line in GitHub web UI, and more than 200 characters in the plaintext markdown file.

Fixes #897

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

File: AUTHORS.md [hunk 1]
```diff
@@ -8,13 +8,12 @@ Manoa.
...
-Department of Earth Sciences, University of Hawai'i at Mānoa, Honolulu, HI, USA
-* [Michael Grund](https://github.com/michaelgrund) | [0000-0001-8759-2018](https://orcid.org/0000-0001-8759-2018) | Innoplexia GmbH, Heidelberg, Germany
-* [Tyler Newton](http://www.tnewton.com/) | [0000-0002-1560-6553](https://orcid.org/0000-0002-1560-6553) | Department of Earth Sciences, University of Oregon, Eugene, OR, USA
-* [Wei Ji Leong](https://github.com/weiji14) | [0000-0003-2354-1988](https://orcid.org/0000-0003-2354-1988) | Antarctic Research Centre, Victoria University of Wellington, Wellington, New Zealand
+* [Dongdong Tian](https://seisman.info/) | [0000-0001-7967-1197](https://orcid.org/0000-0001-7967-1197) | Michigan State University
+* [Leonardo Uieda](http://www.leouieda.com/) | [0000-0001-6123-9515](https://orcid.org/0000-0001-6123-9515) | University of Liverpool
+* [Liam Toney](https://liam.earth/) | [0000-0003-0167-9433](https://orcid.org/0000-0003-0167-9433) | University of Alaska Fairbanks
+* [Malte Ziebarth](https://github.com/mjziebarth) | [0000-0002-5190-4478](https://orcid.org/0000-0002-5190-4478) | GFZ
```
Comment (seisman) [inline AUTHORS.md:14]:
```suggestion
* [Malte Ziebarth](https://github.com/mjziebarth) | [0000-0002-5190-4478](https://orcid.org/0000-0002-5190-4478) | GFZ German Research Centre for Geosciences
```

**Review by seisman — APPROVED** [review]
Looks good to me. Ping @GenericMappingTools/python for second round of review.

**Comment by liamtoney** [comment]
Looks good to me. Thanks @core-man!

**Comment by weiji14** [comment]
> Please open a new pull request to add yourself to the `AUTHORS.md` file. We hope that this was a good experience for you. Let us know if there is any way that the contributing process could be improved.

Thanks @core-man! Definitely add your name to the AUTHORS.md list too when you feel like it :wink:

**Comment by core-man** [comment]
> > Please open a new pull request to add yourself to the `AUTHORS.md` file. We hope that this was a good experience for you. Let us know if there is any way that the contributing process could be improved.
> 
> Thanks @core-man! Definitely add your name to the AUTHORS.md list too when you feel like it

Thanks a lot for the remind @weiji14. I will do it when I am ready to contribute more and regularly to the meaningful project.

**Pull request merged**